### PR TITLE
Self Service BI plugin

### PR DESCRIPTION
**Pull request by SPECUSA**
New plugin for Pentaho CE - Self Service BI

### CODE REVIEW

File: marketplace.xml [hunk 1]
```diff
@@ -7547,6 +7547,49 @@ More on: https://anotherreeshu.wordpress.com/2015/01/13/special-character-remove
...
+  </description>
+  <author>SPEC-INDIA</author>
+  <author_url>http://www.spec-india.com/</author_url>
+  <dependencies>CDA, CCC, Saiku Analytics</dependencies>
+  <forum_url>https://github.com/SPECUSA/SelfServiceBI</forum_url>
+  <cases_url>https://github.com/SPECUSA/SelfServiceBI</cases_url>
+  <license_name>GNU General Public License</license_name>
+  <license_text>For more details see: http://www.gnu.org/licenses/</license_text>
+  <support_level>PROFESSIONALY_SUPPORTED</support_level>
+  <version>
```
Comment (pamval) [inline marketplace.xml]: This is wrong. Version element must be a child element of the Versions element. There's a XSD file in the project, please use that to validate your additions.

File: marketplace.xml [hunk 2]
```diff
@@ -7547,6 +7547,49 @@ More on: https://anotherreeshu.wordpress.com/2015/01/13/special-character-remove
...
+      <branch>STABLE</branch>
+      <version>1.0.0</version>
+      <name>SelfServiceBI</name>
+      <package_url>https://github.com/SPECUSA/SelfServiceBI/raw/master/SelfServiceBI-1.0.0.zip</package_url>
+       <description>
```
Comment (pamval) [inline marketplace.xml]: The zip file is unpacking the wrong structure. It should contain SelfServiceBI/ only and not SelfServiceBI-1.0.0/SelfServiceBI/ as it currently has.

**Comment by pamval** [comment]
Additionally, I see the zip file for the plugin is almost 70mb. Almost al the jars in the lib folder do not need to be there since they are already provided by the platform. Checking your build-time dependencies will certainly reduce the size of the pluign,

**Comment by SPECUSA** [comment]
hello @pamval 
Please review our Self Service BI plugin

**Pull request merged**